### PR TITLE
FeatureFlagFormPropertyMapper.conf.php in default config and `MapLtiS…

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Missing default config for SectionVisibilityFilter and FeatureFlagFormPropertyMapper.conf.php

https://oat-sa.atlassian.net/browse/NEX-1989

### CODE REVIEW

**Comment by siwane** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
LGTM

**Review by boajer — COMMENTED** [review]
Would it be possible to override that settings using seed file? 
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -25,6 +25,7 @@
 use oat\taoLti\controller\Security;
 use oat\taoLti\scripts\install\GenerateKeys;
 use oat\taoLti\scripts\install\InstallServices;
+use oat\taoLti\scripts\install\MapLtiSectionVisibility;
```
Comment (gabrielfs7) [inline manifest.php:28]: tao manifest version needs to be fixed: `'tao' => '>=45.14.0'` is not existent anymore.

**Review by gabrielfs7 — APPROVED** [review]
LGTM. Good job! Thanks for the documentation!

File: scripts/install/MapLtiSectionVisibility.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+namespace oat\taoLti\scripts\install;
+
+use oat\oatbox\extension\AbstractAction;
+use oat\tao\model\menu\SectionVisibilityFilter;
+
+class MapLtiSectionVisibility extends AbstractAction
+{
+    public function __invoke($params)
+    {
+        $this->getServiceManager()->register(
```
Comment (mike-ionut-mihai-sandu-tao) [inline scripts/install/MapLtiSectionVisibility.php:30]: i think this should inside a migration and not an install script

File: scripts/install/MapLtiSectionVisibility.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+namespace oat\taoLti\scripts\install;
+
+use oat\oatbox\extension\AbstractAction;
+use oat\tao\model\menu\SectionVisibilityFilter;
+
+class MapLtiSectionVisibility extends AbstractAction
+{
+    public function __invoke($params)
```
Comment (mike-ionut-mihai-sandu-tao) [inline scripts/install/MapLtiSectionVisibility.php:28]: no test for new class

File: scripts/install/MapLtiSectionVisibility.php (hunk 2 above)
Comment (gabrielfs7) [reply scripts/install/MapLtiSectionVisibility.php:30]: Unfortunately it does not run migrations on install :/

Comment (bartlomiejmarszal) [reply scripts/install/MapLtiSectionVisibility.php:30]: This class is providing default configuration for new instance. Migrations are used only for  `taoUpdate`

File: scripts/install/MapLtiSectionVisibility.php (hunk 3 above)
Comment (bartlomiejmarszal) [reply scripts/install/MapLtiSectionVisibility.php:28]: This is an install script. I think this unit test is useless.

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**